apps/ui/src/common/app.scss

```diff
@@ -14,3 +14,7 @@ dl {
   margin-right: max(30px, calc(15px + env(safe-area-inset-right))) !important;
   margin-left: max(30px, calc(15px + env(safe-area-inset-left))) !important;
 }
+
```

Comment:
as above - Part of another (unapproved) PR. I don't think this should be included.